packages/scaffolding-cli/templates/src/ssr/lib/contentful-api.ts

```diff
@@ -0,0 +1,81 @@
+import {createClient} from "contentful"
+
+const client = createClient({
+    space: process.env.NEXT_PUBLIC_CONTENTFUL_SPACE_ID,
+    accessToken: process.env.NEXT_PUBLIC_CONTENTFUL_ACCESS_TOKEN,
```

Comment:
Why are these referenced here with no default, should we not pull them in from env config?